doc/architecture/decisions/0002-refactor-functional-tests-into-surveys-components-and-features.md

```diff
@@ -17,10 +17,6 @@ The functional tests should be organised in such a way that encourages consisten
 tests/
   functional/
     spec/
-        surveys/
-            census-household.spec.js
-            mci.spec.js
-            ...
```

Comment:
wondering if this should be put back? I don't think historical descision docs should be updated

Reply:
Reverted back to the original 👍

Reply:
Same case as the primary person file we talked about before, don't change historical docs.